causalml/feature_selection/filters.py

```diff
@@ -0,0 +1,370 @@
+"""
+Filter feature selection methods for uplift modeling
+
+- Currently only for classification problem: the outcome variable of uplift model is binary.
+"""
+
+import numpy as np
+import pandas as pd
+import statsmodels.api as sm
+from scipy import stats
+    
+class FilterSelect:
+    """A class for feature importance methods.
+    """
+
+    def __init__(self):
+        return
+
+    def _convert_treatment(self, data, T_name_before, treatment_key):
+        """
+        Convert text treatment labels into 0 (control) and 1 (treated).
```

Comment:
Could we add the args here?

Reply:
This is an intermediate function that is not user-facing. It seems sklearn skips the args for intermediate functions (with _ prefix), for example: https://github.com/scikit-learn/scikit-learn/blob/b194674c4/sklearn/ensemble/_forest.py#L2172. But we can make our own rules. If we decide to add args for all functions, I will add args here.

Reply:
I'm also fine with the convention of not specifying the args for non-public functions. I was purely thinking it from the point of view that it was not immediately clear to me what "T_name_before" means.

Reply:
I second to @t-tte. When it might not be straightforward to others, it's better to have docstrings.